tidb-binlog/tidb-binlog-overview.md

```diff
@@ -15,6 +15,10 @@ TiDB Binlog has the following features:
 * **Data replication:** replicate the data in the TiDB cluster to other databases
 * **Real-time backup and restoration:** back up the data in the TiDB cluster and restore the TiDB cluster when the cluster fails
 
+> **Note:**
+>
+> TiDB Binlog is not compatible with some features introduced in TiDB v5.0.0-rc and they cannot be used together. For details, see [Notes](#notes). It is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead.
```

Comment:
```suggestion
> TiDB Binlog is not compatible with some features introduced in TiDB v5.0.0-rc, and they cannot be used together. For details, see [Notes](#notes). It is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead of TiDB Binlog.
```
Let's make it clear that we are talking about "Binlog" rather than "features introduced in 5.0 rc"